Samples/StyleTransfer/AppModel.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Media;
+using Windows.Media.Capture;
+using Windows.Media.Core;
+using Windows.Media.Playback;
+using Windows.UI.Composition;
+using Windows.UI.Xaml.Controls;
+
+namespace StyleTransfer
+{
+    class AppModel : INotifyPropertyChanged
+    {
+        public AppModel()
+        {
+            this._modelSource = "candy";
+            this._selectedCameraIndex = 0;
+            this._outputCaptureElement = new CaptureElement();
+        }
+
+        // Name of the style transfer model to apply to input media
+        private string _modelSource;
+        public string ModelSource
+        {
+            get { return _modelSource; }
+            set
+            {
+                _modelSource = value;
+                OnPropertyChanged();
+            }
+        }
+
+        // Type of input media to use
+        private string _inputMedia;

```

Comment:
>inputMedia [](start = 24, length = 10)

Why string? Cant this be an enum?

Here is a UWP example of using enums for command parameters
<Button Command="{Binding CheckWeatherCommand}">
  <Button.CommandParameter>
     <local:WeatherEnum>Cold</local:WeatherEnum>
  <Button.CommandParameter>
</Button>

Reply:
Changed to enum